src/notification/notification.stories.ts

```diff
@@ -233,29 +233,51 @@ storiesOf("Components|Notification", module)
 	}))
 	.add("With custom content", () => ({
 		template: `
-			<ibm-toast [notificationObj]="{type: 'error'}">
-				<h3 class="bx--toast-notification__title">Hello</h3>
-				<p class="bx--toast-notification__subtitle">
-					Lorem ipsum dolor sit amet, consectetur adipiscing elit.
-					Ut scelerisque, risus eu tincidunt faucibus
-				</p>
+			<ibm-toast [notificationObj]="{
+				type: 'error',
+				title: 'Sample custom toast',
+				subtitle: 'Sample subtitle message',
+				caption: 'Sample caption',
+				lowContrast: lowContrast,
+				content: customToastContent,
```

Comment:
Going to be picky - searching through the codebase we have one existing instance of `obj.content` (in the dialog) and several instances of `obj.template` ... I'm going to suggest that we go with `obj.template` to _try_ and keep things consistent

Reply:
also, do we want to keep the default layout when a template is used? It's possible a user would want full control over the content portion of a notification ... we can pass the `notificationObj` through as template context